starknet_devnet/blueprints/rpc.py

```diff
@@ -235,6 +276,13 @@ async def call(contract_address: str, entry_point_selector: str, calldata: list,
         raise RpcError(code=-1, message=ex.message) from ex
 
 
+async def estimate_fee():
+    """
+    Get the estimate fee for the transaction
+    """
+    raise NotImplementedError()
```

Comment:
No particular reason, I just didn't do it in this PR but wanted to add a signature for this metod. Current RPC implementation in devnet does not support adding transactions at all so estimating fees isn't all that useful on it's own.

Perhaps this could be added in separate PR along with the rpc write api?

Reply:
ok, makes sense